jetty-core/jetty-deploy/src/main/config/etc/jetty-deployment-scanner.xml

```diff
@@ -39,4 +39,11 @@
   </Set>
   <Set name="deferInitialScan" property="jetty.deploy.deferInitialScan" />
   <Set name="scanInterval" property="jetty.deploy.scanInterval" />
+  <Ref refid="deploymentManager">
+    <Call name="addBean">
+      <Arg>
+        <Ref refid="deploymentScanner" />
+      </Arg>
+    </Call>
+  </Ref>
```

Comment:
I know this fixes the problem, but it is still strange that the scanner has-a deployer (passed in the constructor) and the deployer now has-a scanner (as a bean).

However, this change is essentially what my branch does.. except I have some renames in it and by default I add the scanner as a bean to the server, as the scanner is able to create a default simple deployer if it doesn't find one as a bean (or is not passed one in the constructor).